src/game-ratings/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Game Ratings
+---
+
+# What is voltage?
+
+Voltage is Game Jolt's community-driven, weighted rating system. Voltage is a measure of game's quality and popularity.
+
+Voltage is represented by the voltmeter icon near the top right of game pages.
```

Comment:
This is incorrect. The meters are literally JUST averages now. There's no way to see the actual voltage score.